grammars/language-asciidoc.cson

```diff
@@ -1221,6 +1224,14 @@ repository:
         end: "^(\\1)$"
       }
     ]
+  "literal-paragraph":
+    patterns: [
+      {
+        name: "markup.block.literal.asciidoc"
+        begin: "^\\p{Blank}+(.*)$"
```

Comment:
Were you thinking of more changes other than `.+` to `.*` ? Otherwise it seems completing this enhancement would be relatively easy, as you already did most of the work.